dags/genie-nf-validate.py

```diff
@@ -0,0 +1,46 @@
+from datetime import datetime
+
+from airflow.decorators import dag, task
+
+from dag_content.nextflow_tower_functions import create_and_open_tower_workspace
+
+
+@dag(
+    schedule_interval="@daily",
+    start_date=datetime(2023, 2, 9),
+    catchup=False,
+    default_args={
+        "retries": 1,
+    },
+    tags=["nextflow_tower"],
+)
+def genie_nf_validate_dag():
+    @task()
+    def launch_tower_workflow(workspace_id: str):
+        """
+        Launches tower workflow
+
+        Args:
+            workspace_id (str): Workspace ID for tower run
+        """
+        tower_utils = create_and_open_tower_workspace(
+            tower_access_token="TOWER_ACCESS_TOKEN_GENIE", platform="sage-dev", workspace_id=workspace_id
+        )
+        tower_utils.launch_workflow(
+            compute_env_id="1IEIFEJSVdDzQhwQlZRrHW",
+            pipeline="Sage-Bionetworks-Workflows/nf-genie",
+            run_name="airflow-genie-validate-test",
```

Comment:
One thing I will say is that are we going to have "prod" and "test" DAGs?  Or should the DAG just be configurable to launch the production validation job?

Reply:
Ideally, there is no notion of test/dev vs prod in the DAGs. The variable bits between test/dev and prod should be stored in parameters or connections.